angular/lib/src/components/slider/slider.component.ts

```diff
@@ -1,115 +1,210 @@
-import type {SliderWidget} from '@agnos-ui/angular-headless';
+import type {SliderContext, SliderProps, SliderWidget, SlotContent} from '@agnos-ui/angular-headless';
 import {
 	BaseWidgetDirective,
+	ComponentTemplate,
+	SlotDefaultDirective,
+	SlotDirective,
 	UseDirective,
 	auBooleanAttribute,
 	auNumberAttribute,
 	callWidgetFactory,
 	createSlider,
 	useDirectiveForHost,
 } from '@agnos-ui/angular-headless';
-import {ChangeDetectionStrategy, Component, EventEmitter, Input, NgZone, Output, ViewEncapsulation, forwardRef, inject} from '@angular/core';
+import {type WritableSignal, writable} from '@amadeus-it-group/tansu';
+import {NgClass} from '@angular/common';
+import type {AfterContentChecked} from '@angular/core';
+import {
+	ChangeDetectionStrategy,
+	Component,
+	ContentChild,
+	Directive,
+	EventEmitter,
+	Input,
+	NgZone,
+	Output,
+	TemplateRef,
+	ViewChild,
+	ViewEncapsulation,
+	forwardRef,
+	inject,
+} from '@angular/core';
 import {NG_VALUE_ACCESSOR} from '@angular/forms';
 import {take} from 'rxjs';
 
+@Directive({selector: 'ng-template[auSliderLabel]', standalone: true})
+export class SliderLabelDirective {
+	public templateRef = inject(TemplateRef<SliderContext>);
+	static ngTemplateContextGuard(_dir: SliderLabelDirective, context: unknown): context is SliderContext & {value: number} {
+		return true;
+	}
+}
+
+@Directive({selector: 'ng-template[auSliderHandle]', standalone: true})
+export class SliderHandleDirective {
+	public templateRef = inject(TemplateRef<SliderContext>);
+	static ngTemplateContextGuard(
+		_dir: SliderHandleDirective,
+		context: unknown,
+	): context is SliderContext & {item: {value: number; id: number; ariaLabel: string}} {
```

Comment:
minor (if you don't change it I will add it in aria-valuetext PR): could be replaced by new interface (SliderHandle) in core

Reply:
Done, had to create also the corresponding `AdaptSlotContentProps` to match interfaces